src/ios/PurchasesPlugin.swift

```diff
@@ -186,48 +191,42 @@ public class CDVPurchasesPlugin : CDVPlugin {
             return
         }
 
-        RCCommonFunctionality.simulatesAskToBuyInSandbox = askToBuyInSandbox
+        CommonFunctionality.simulatesAskToBuyInSandbox = askToBuyInSandbox
         self.sendOKFor(command: command)
     }
 
     @objc(isAnonymous:)
     func isAnonymous(command: CDVInvokedUrlCommand) {
-        let result = CDVPluginResult(status: .ok, messageAs: RCCommonFunctionality.isAnonymous())
+        let result = CDVPluginResult(status: .ok, messageAs: CommonFunctionality.isAnonymous)
         self.commandDelegate.send(result, callbackId: command.callbackId)
     }
 
-    // NOTE: This method crashes now that it's in Swift. The underlying method call relies on an implicit cast
```

Comment:
This is actually still a problem, just slightly different problem now: https://github.com/RevenueCat/purchases-hybrid-common/pull/159